telegram-desktop-bin/lure.sh

```diff
@@ -0,0 +1,69 @@
+name='telegram-desktop-bin'
+version='4.5.2'
+release='1'
+desc='Official desktop version of Telegram messaging app - Static binaries'
+homepage='https://github.com/telegramdesktop/tdesktop'
+maintainer="Евгений Храмов <xpamych@yandex.ru>"
+architectures=('amd64')
+license=('GPLv3')
+provides=('telegram-desktop')
+conflicts=('telegram-desktop')
+deps=('desktop-file-utils' 'glib2' 'hicolor-icon-theme' 'dbus-libs' 'libX11' 'libappindicator-gtk3' 'xdg-utils')
+build_deps=('chrpath')
+
+sources=(
+	"${homepage}/raw/master/Telegram/Resources/art/icon16.png"
+	"${homepage}/raw/master/Telegram/Resources/art/icon32.png"
+	"${homepage}/raw/master/Telegram/Resources/art/icon48.png"
+	"${homepage}/raw/master/Telegram/Resources/art/icon64.png"
+	"${homepage}/raw/master/Telegram/Resources/art/icon128.png"
+	"${homepage}/raw/master/Telegram/Resources/art/icon256.png"
+	"${homepage}/raw/master/Telegram/Resources/art/icon512.png"
+	"${homepage}/releases/download/v${version}/tsetup.${version}.tar.xz")
+checksums=(
+	'SKIP'
+	'SKIP'
+	'SKIP'
+	'SKIP'
+	'SKIP'
+	'SKIP'
+	'SKIP'
+	'SKIP'
+)
+
+package() {
+
+	cd "${srcdir}/"
+
+	# Creating needed directories
+	install -dm755 "${pkgdir}/usr/bin"
+	install -dm755 "${pkgdir}/usr/share/pixmaps/"
+	install -dm755 "${pkgdir}/usr/share/applications/"
```

Comment:
Just use `mkdir -p` if you're only creating directories.

```suggestion
	mkdir -p "${pkgdir}/usr/bin"
	mkdir -p "${pkgdir}/usr/share/pixmaps/"
	mkdir -p "${pkgdir}/usr/share/applications"
```